stable/rabbitmq/values.yaml

```diff
@@ -146,6 +146,13 @@ rabbitmq:
   ## Configuration file content: advanced configuration
   ## Use this as additional configuraton in classic config format (Erlang term configuration format)
   advancedConfiguration: |-
+    # If you set LDAP with TLS/SSL enabled and you are using self-signed certificates, uncomment these lines.
+    #[
+    #{rabbitmq_auth_backend_ldap, [
+    #{ssl_options, [{verify, verify_none},
+    #{fail_if_no_peer_cert, false}]}
+    #]}
+    #].
```

Comment:
You're including this commented block even when LDAP is not enabled. I would use:
```
  ## Configuration file content: advanced configuration
  ## Use this as additional configuraton in classic config format (Erlang term configuration format)
  ## Example for LDAP with TLS/SSL enabled and self-signed certificates:
  ## advancedConfiguration: |-
  ## [{
  ##     rabbitmq_auth_backend_ldap, 
  ##     [{
  ##         ssl_options, 
  ##         [{
  ##             verify, verify_none
  ##         }, {
  ##             fail_if_no_peer_cert, 
  ##             false
  ##         }]
  ##     ]}
  ## }]
  ##
  # advancedConfiguration: 
```